.github/workflows/test_published_images.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         variant: 
           - ''
           - 'slim'
-        distribution_type: ${{ fromJson(needs.set-matrix.outputs.distribution-type-matrix) }}
+        distribution-type: ${{ fromJson(needs.set-matrix.outputs.distribution-type-matrix) }}
```

Comment:
why this rename?

Reply:
> why this rename?

Consistency between the matrix name and its source. Probably not required but neater. Can revert if problematic…